### PR TITLE
Debug Off: Media Files Fail to Render in Production

### LINKED ISSUE #19: Media files not rendered in Production with Debug off.
Currently, whitenoise is set up to render media files in production but it isn't working.

### PR DESCRIPTION
**Pull request by aybruhm**
Image uploads from the admin dashboard to cloudinary are rendered when debug is set to False.

This PR resolves #19

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+DJANGO_ALLOWED_HOSTS="0.0.0.0,127.0.0.1"
```
Comment (devgenix) [inline .env.template:1]: Can you please remove Airtable environmental Variables?, we no longer use Airtable in the project.

**Review by devgenix — CHANGES_REQUESTED** [review]

File: .github/workflows/django.yml
Comment (devgenix) [inline .github/workflows/django.yml:1]: Remove it from here too.

**Review by devgenix — CHANGES_REQUESTED** [review]

File: .github/workflows/django.yml
Comment (aybruhm) [reply .github/workflows/django.yml:1]: If I do, the test cases would fail.

File: .env.template (hunk 1 above)
Comment (aybruhm) [reply .env.template:1]: Oh, is that so? Very well, then.

**Review by devgenix — CHANGES_REQUESTED** [review]
The url returned in the image should be the link to the image on cloudinary. 

image

**Comment by aybruhm** [comment]
> The url returned in the image should be the link to the image on cloudinary.
> 
> image

Kindly pull and check again.

**Comment by devgenix** [comment]
> > The url returned in the image should be the link to the image on cloudinary.
> > image
> 
> Kindly pull and check again.

Agba Idan. 
As soon as I requested it, I saw your last commit.

**Review by devgenix — APPROVED** [review]

**Pull request merged**